app/Console/Commands/ACL/SynchroniseRolesAndPermissions.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands\ACL;
+
+use App\ACL\Contracts\ACLService;
+use Illuminate\Console\Command;
+
+class SynchroniseRolesAndPermissions extends Command
```

Comment:
Wil je dat ik deze commando ook elke keer bij deployment uitvoer?

Reply:
This should indeed be added to the deployment steps of acceptance. I've created a ticket for you: https://github.com/KA-Huis/space-management/issues/91